base/ryu/utils.jl

```diff
@@ -0,0 +1,391 @@
+const MANTISSA_MASK = 0x000fffffffffffff
+const EXP_MASK = 0x00000000000007ff
+
+memcpy(d, doff, s, soff, n) = ccall(:memcpy, Cvoid, (Ptr{UInt8}, Ptr{UInt8}, Int), d + doff - 1, s + soff - 1, n)
```

Comment:
wrong method signature for memcpy and memmove—also I think this is already `unsafe_copyto!`